lib/browser/default-menu.ts

```diff
@@ -1,12 +1,16 @@
 import { app, Menu } from 'electron/main';
 import { shell } from 'electron/common';
 
-const v8Util = process._linkedBinding('electron_common_v8_util');
-
 const isMac = process.platform === 'darwin';
 
+let applicationMenuSet = false;
```

Comment:
nit: let's take the opportunity to improve the name slightly here
```suggestion
let applicationMenuWasSet = false;
```

& related renames.